### PR TITLE
Calcium phosphate

### PR DESCRIPTION
**Pull request by Rocket-F-1024**
#### Summary

Content "Add calcium phosphate and related recipes"

#### Purpose of change

Based on a discussion in another PR(https://github.com/CleverRaven/Cataclysm-DDA/pull/74997).

#### Describe the solution

Add calcium phosphate.
Add recipes that use bones to produce calcium phosphate.
Changing the raw material for the recipe to produce white phosphorus from bone meal to calcium phosphate.
Modify the data and related recipes for white phosphorus.

#### Describe alternatives you've considered

None.

#### Testing

Go into the game, check the recipe and item, and all is well.

#### Additional context

None,

### CODE REVIEW

**Comment by Maleclypse** [comment]
Can you link to recipes from journals or other places that list the actual equipment needed and amount of heat applied etc so I can review the new recipes compared to that?  That will make reviewing this much faster.

File: data/json/recipes/recipe_medsandchemicals.json [hunk 1]
```diff
@@ -1535,12 +1535,60 @@
...
+    "charges": 1,
+    "batch_time_factors": [ 95, 1 ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "difficulty": 3,
+    "skill_used": "chemistry",
+    "time": "10 m",
+    "qualities": [ { "id": "CHEM", "level": 2 } ],
+    "book_learn": [ [ "textbook_gaswarfare", 2 ], [ "textbook_chemistry", 3 ], [ "adv_chemistry", 3 ], [ "textbook_anarch", 3 ] ],
+    "tools": [ [ [ "surface_heat", 30, "LIST" ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ],
```
Comment (kevingranade) [inline data/json/recipes/recipe_medsandchemicals.json:1583]: This needs distillation apparatus and several other pieces of equipment, please do some research here.

File: data/json/recipes/recipe_medsandchemicals.json [hunk 2]
```diff
@@ -1535,12 +1535,60 @@
...
+    "charges": 8,
+    "batch_time_factors": [ 95, 1 ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "difficulty": 3,
+    "skill_used": "chemistry",
+    "time": "10 m",
+    "qualities": [ { "id": "CHEM", "level": 2 } ],
+    "book_learn": [ [ "textbook_gaswarfare", 2 ], [ "textbook_chemistry", 3 ], [ "adv_chemistry", 3 ], [ "textbook_anarch", 3 ] ],
+    "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
```
Comment (kevingranade) [inline data/json/recipes/recipe_medsandchemicals.json:1551]: What process is this supposed to be? I'm not aware of a process where heating bone meal on a stove is productive.

**Review by kevingranade — CHANGES_REQUESTED** [review]
This is very abbreviated compared to what even wikipedia outlines, please look into what is entailed, both in tools and components in some actual production methods.

**Comment by Night-Pryanik** [comment]
Closing as stale. If you wish to continue working on this, ping me to reopen.

**Pull request closed without merging**